### ISSUE TITLE
Does it gem support Embedded Payments ?

### ISSUE THREAD

**Issue by sirius248** [lemboy/spree_paypal_adaptive#1]

**Comment by lemboy** [comment]
At this stage there is still no. Sorry.
But it is quite possible to do in the near future, I think.

**Comment by sirius248** [comment]
What is your suggestion if I use your gem add that feature ?

**Comment by lemboy** [comment]
I had a little misunderstanding. You want to add this feature  yourself?

**Comment by sirius248** [comment]
Yeah. I mean if I add that feature myself, so do you have any idea that could help me do it

**Comment by lemboy** [comment]
Just now I have only [this](https://developer.paypal.com/docs/classic/adaptive-payments/ht_ap-embeddedPayment-curl-etc/) advice about subject. But I'm interested to add this feature, so, we can try to do this.

**Comment by sirius248** [comment]
Ok. I'm reading up that documentation. If I find something, I will create pull request on your repo

**Comment by lemboy** [comment]
Fine. And I'm will look to problem in the next two days. Good luck!

**Comment by sirius248** [comment]
I make some move. However I got the following error when I tried to do `bundle exec rails g spree_paypal_adaptive:install`

```
#<Class:0x0000010a5b11d8> (NoMethodError)
    from /Users/longn/.rvm/gems/ruby-2.1.1/bundler/gems/spree_paypal_adaptive-0e9393ed68b3/app/models/spree/payment_decorator.rb:4:in `block in <module:Spree>
```

Don't know `durably_decorate` is for ?

**Comment by lemboy** [comment]
What versions of spree/dropship you are using?

**Edit.** 
[durable_decorator](https://github.com/jumph4x/durable_decorator) used  [here](https://github.com/lemboy/spree_paypal_adaptive/blob/master/app/models/spree/payment_decorator.rb) because Spree not allow multiply payments by single order and invalidate other payments if you add new one.

**Comment by sirius248** [comment]
I got the embedded payment work on my case now. However I cloned your gem, and customize it a little bit. Since I was using spree 2.3.4 Thank for the gem though it does save my time.
